Databricks/Alluxio-2.9.0-without-CVE/libexec/alluxio-config.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# Included in all the Alluxio scripts with source command should not be executable directly also
+# should not be passed any arguments, since we need original $*
```

Comment:
why is this file here?  description of changes.

Reply:
See the comments in the `build-from-src-and-fix-CVE.sh`
```
# update `libexec/alluxio-config.sh`, replace the client and server jar as the Alluxio building script does.
# For details, refer to:
# https://github.com/Alluxio/alluxio/blob/v2.9.0/dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go#L274-L275
git am /tmp/Alluxio-2.9.0-without-CVE/update-assembly-jar-path.patch
```